timers.js

```diff
@@ -1,4 +1,28 @@
+var start = document.getElementById("start");
+var timer = document.getElementById("timer");
+var reset = document.getElementById("reset");
+var pause = document.getElementById("pause");
 
+var seconds = 0;
+var timerID;
+
+start.addEventListener("press", function() {
```

Comment:
so "press" is not really an event, you could either have "keypress" and specify the key you want, or , "click" . code works perfectly otherwise, great job! ~